# No blockless switch statements
## Rule Details
Always use brackets for `switch` statements

Example of **correct** code for this rule:

```js
  switch (green) {
    case 'orange': {
      color = colors.purple
      break
    }
    case 'blue': {
      color = colors.black
      break
    }
    default: {
      color = colors.white
      break
    }
  }
```
Example of **incorrect** code for this rule:
```js
   switch (green) {
     case 'orange':
       color = colors.purple
       break
     default: color = colors.white
   }
```
