# No blockless if statements
## Rule Details
Always use brackets for `if` statements

Example of **correct** code for this rule:

```js
  if (baz) {
    baz()
  } else {
    bar()
  }
```
Example of **incorrect** code for this rule:
```js
  if (foo && bar)
    foo()
    bar()
```
