# No unnecessary ternary expressions
## Rule Details
Avoid using ternary expressions when a simpler logical expression is feasible.

Example of **correct** code for this rule:

```js
  let x = condition ? false : value
```
Example of **incorrect** code for this rule:
```js
  let x = condition ? true : value
```
can be written as a logical expression:
```js
let x = Boolean(condition) || value
```
