# No property change in update lifecycle method
## Rule Details
Disallows property changes in the `update` lifecycle method
Example of **correct** code for this rule:

```js
  class Foo extends LitElement {
    static get properties() {
        return { prop: { type: Number } };
    }
    update() {
        super.update();
        this.prop2 = 5;
    }
}
```
Example of **incorrect** code for this rule:
```js
  class Foo extends LitElement {
    static get properties() {
        return { prop: { type: String } };
    }
    update() {
        super.update();
        this.prop = 'foo';
    }
}
```
